File: src/main/java/alfheim/common/spell/wind/SpellWindBlade.java
package alfheim.common.spell.wind;

import alfheim.api.entity.EnumRace;
import alfheim.api.spell.SpellBase;
import alfheim.common.entity.spell.EntitySpellWindBlade;
import net.minecraft.entity.EntityLivingBase;

public class SpellWindBlade extends SpellBase {

	public SpellWindBlade() {
		super("windbalde", EnumRace.SYLPH, 8000, 120, 10);
	}

	@Override
	public SpellCastResult performCast(EntityLivingBase caster) {
		SpellCastResult result = checkCast(caster);
		if (result == SpellCastResult.OK) caster.worldObj.spawnEntityInWorld(new EntitySpellWindBlade(caster.worldObj, caster));
		return result;
	}
}

File: src/main/java/alfheim/common/spell/wind/SpellWindBlade.kt
package alfheim.common.spell.wind

import alfheim.api.entity.EnumRace
import alfheim.api.spell.SpellBase
import alfheim.common.entity.spell.EntitySpellWindBlade
import net.minecraft.entity.EntityLivingBase

class SpellWindBlade: SpellBase("windbalde", EnumRace.SYLPH, 8000, 120, 10) {
	
	override fun performCast(caster: EntityLivingBase): SpellBase.SpellCastResult {
		val result = checkCastOver(caster)
		if (result == SpellBase.SpellCastResult.OK) caster.worldObj.spawnEntityInWorld(EntitySpellWindBlade(caster.worldObj, caster))
		return result
	}
}